core/storage/base_model/gae_models.py

```diff
@@ -57,6 +57,13 @@
     'NOT_APPLICABLE'
 )
 
+EXPORT_METHOD = utils.create_enum(  # pylint: disable=invalid-name
+    'SINGLE_UNSHARED_INSTANCE',
```

Comment:
Add a comment above each item here to describe clearly what it means. Follow the example in DELETION_POLICY above.

Reply:
Done